dpc/src/circuits/tests.rs

```diff
@@ -150,7 +150,7 @@ mod testnet1 {
 
     #[test]
     fn test_dpc_execute_circuits() {
-        dpc_execute_circuits_test::<Testnet1>(274298, 147941);
+        dpc_execute_circuits_test::<Testnet1>(274316, 147941);
```

Comment:
Testnet1's inner circuit changes with this PR. Is that okay? It's a necessary soundness fix.